docs/backends/Google.md

```diff
@@ -262,3 +262,36 @@ This filesystem has two required configuration options:
   documentation](https://www.ncbi.nlm.nih.gov/books/NBK63512/#Download.are_downloaded_files_encrypted)
   for more information on obtaining your NGC.  The `ngc` value provided above
   is the sample credential file.
+
+### Virtual Private Network
+
+In PAPIV2, to run your jobs in private network add the `virtual-private-cloud` stanza in papiv2 section in `backend` stanza.
+
+```
+backend {
+  ...
+  providers {
+  	...
+  	PapiV2 {
+  	  actor-factory = "cromwell.backend.google.pipelines.v2alpha1.PipelinesApiLifecycleActorFactory"
+  	  config {
+  		...
+  		virtual-private-cloud {
+  	          network-label-key = "my-private-network"
+  	          auth = "reference-to-auth-scheme"
+  	        }
+  	    ...
+  	  }  
+      }
+  }
+}
+```
+
+
+The `network-label-key` should reference the key in the label in your project, whose value is the name of your private network. And the `auth` should reference to an auth scheme in `google` stanza which should be used to request Google Cloud to get the project's metadata. For example, if your `virtual-private-cloud` config looks like the one above, and one of the labels in your project is
+
+```
+"my-private-network" = "vpc-network"
+```
+
+Cromwell will get labels from the project's metadata and look for a label whose key is `my-private-network`. Then, it will use the value of the label, which is `vpc-network` here, as the name of private network and run the jobs on this network.
```

Comment:
What happens if the key isn't present in metadata?

Reply:
It will fail with an error `Unable to extract labels containing network information. Project metadata does not have network label key <key>` .

Reply:
Could we add that to the documentation?

Reply:
Sure. I will add that.